frontend/src/types/Round.ts

```diff
@@ -0,0 +1,7 @@
+import { Action } from "./Action";
+
+export type Round = Action[];
```

Comment:
This may seem a bit weird, but since the prop is called `next_round`, I consider the array of actions a "round" and therefore type it as such. Perhaps we can think about better naming if this is too confusing. For example, we could also rename (both in the frontend and backend) `next_round` to `actions` and then the `next_round` http request will respond with an object like this:

```json
{
  "actions": [{...}, {...}, {...}]
}
```

which might make more sense.

Reply:
I'm not too sure about this change. It is indeed weird that a "round" refers to an array of actions - at the same time, it does make sense that a participant has to take a number of actions within a round. If we document it, I don't think it's too confusing to keep handling *round* / *next_round* references. If we were to change the name of the API call, I would argue that would mean we'd also have to change the name of the `next_round` function in all the rule files, and all other references to `round` in the backend. Plus, referring to `currentActions` (i.e., the array of actions last retrieved from the backend) vs. `currentAction` (i.e., the view that's presented to the user right now) would be confusing, too.

Reply:
Ah, but I would not change the name of the api call. You request the next round, it gives you back an object that represents the next round, and that object contains actions.